version/news/NEWS-2024.08.0-cranberry-hibiscus.md

```diff
@@ -95,6 +96,13 @@
 - Fixed bug that prevented Workbench from respecting licenses with an unlimited named user limit when re-activating SCIM provisioned users (rstudio-pro#6561)
 - Fixed bug where user provisioning would not work properly if the homedir column in the database for any user was null (rstudio-pro#6587)
 - Fixed a pagination issue in the SCIM API that could prevent more than 100 users from being provisioned via Okta (rstudio-pro#6608)
+- Fixed various bugs that could cause NSS requests to fail when automatic user provisioning is enabled (rstudio-pro#6676, rstudio-pro#6657)
+
+### Upgrade Instructions
```

Comment:
do you mind adding this to the admin guide as well? https://docs.posit.co/ide/server-pro/2024.08.0/upgrade/upgrade.html#upgrade-requirements